config.json.example

```diff
@@ -127,6 +127,7 @@
         "plantuml":
         {
           "server": "https://www.plantuml.com/plantuml"
-        }
+        },
+        "linkifyHeaderStyle": "gfm"
```

Comment:
Hi @hoijui 
Could we put the available options of this settings in the comment?
I think people will edit this file for their first setup config.
Like:

```suggestion
        "linkifyHeaderStyle": "keep-case" // "keep-case" or "gfm"
```

Reply:
I also didn't know, but here are [no comments in JSON](https://stackoverflow.com/questions/244777/can-comments-be-used-in-json) :/

Reply:
Oh, you're right!

maybe this, but I don't know if people can understand this:
```suggestion
        "//linkifyHeaderStyle": "keep-case or gfm"
```

Reply:
I think maybe we can just remove this in the config.json.example.
It's better to add to the documentation: https://hackmd.io/c/codimd-documentation/%2Fs%2Fcodimd-configuration
We can help to do so.

Reply:
The idea was, to have it in there so it would be used by new installations, but not by old ones.
This way, we could make all new installations adhere to the standard, as used everywhere else.

Reply:
got you